CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.2.10
+- Fix issue where setting a URL when overriding `transform_path` would get a `/` preceding the `http://`
```

Comment:
Change "would get a `/` preceding the `http://`"
To: "results in a `\` character preceding the `http://`"